src/styles/tokens.css

```
@@ -0,0 +1,140 @@
+/* ============================================================
+   Rustatian v2 — design tokens
+   Dark-first, ANSI-accent, technical-not-costume-y.
+   Scheme toggled via [data-theme="dark"] / [data-theme="light"]
+   on <html>.
+   ============================================================ */
+
+@import url('https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap'" (import-notation)

(import-notation)